airflow/contrib/example_dags/example_gcp_transfer.py

```diff
@@ -0,0 +1,276 @@
+    GcpTransferServiceJobDeleteOperator,
+    GcpTransferServiceJobUpdateOperator,
+    GcpTransferServiceOperationsListOperator,
+    GcpTransferServiceOperationGetOperator,
+    GcpTransferServiceOperationPauseOperator,
+    GcpTransferServiceOperationResumeOperator,
+    GcpTransferServiceOperationCancelOperator)
+from airflow.contrib.sensors.gcp_transfer_sensor import GCPTransferServiceWaitForJobStatusSensor
+from airflow.utils.dates import days_ago
+
+# [START howto_operator_gct_common_variables]
+GCP_PROJECT_ID = os.environ.get('GCP_PROJECT_ID', 'example-project')
+GCP_DESCRIPTION = os.environ.get('GCP_DESCRIPTION', 'description')
+GCP_TRANSFER_TARGET_BUCKET = os.environ.get('GCP_TRANSFER_TARGET_BUCKET')
+WAIT_FOR_OPERATION_POKE_INTERVAL = os.environ.get('WAIT_FOR_OPERATION_POKE_INTERVAL', 5)
+
+GCP_TRANSFER_SOURCE_AWS_BUCKET = os.environ.get('GCP_TRANSFER_SOURCE_AWS_BUCKET')
+GCP_TRANSFER_FIRST_TARGET_BUCKET = os.environ.get('GCP_TRANSFER_FIRST_TARGET_BUCKET',
+                                                  'gcp-transfer-first-target')
+GCP_TRANSFER_SECOND_TARGET_BUCKET = os.environ.get('GCP_TRANSFER_SECOND_TARGET_BUCKET',
+                                                   'gcp-transfer-second-target')
+# [END howto_operator_gct_common_variables]
+
+# [START howto_operator_gct_create_job_body_aws]
+create_body_aws = {
+    "description": GCP_DESCRIPTION,
+    "status": "ENABLED",
+    "projectId": GCP_PROJECT_ID,
+    "schedule": {
+        "scheduleStartDate": datetime(2015, 1, 1).date(),
+        "scheduleEndDate": datetime(2020, 1, 1).date(),
+        "startTimeOfDay": datetime.utcnow() + timedelta(minutes=2)
+    },
+    "transferSpec": {
+        'awsS3DataSource': {
+            'bucketName': GCP_TRANSFER_SOURCE_AWS_BUCKET
+        },
+        "gcsDataSink": {
+            "bucketName": GCP_TRANSFER_FIRST_TARGET_BUCKET
+        },
+        "transferOptions": {
+            "overwriteObjectsAlreadyExistingInSink": True
+        },
+    }
+}
+# [END howto_operator_gct_create_job_body_aws]
+
+# [START howto_operator_gct_create_job_body_gcp]
+create_body_gcs = {
+    "description": GCP_DESCRIPTION,
+    "status": "ENABLED",
+    "projectId": GCP_PROJECT_ID,
+    "schedule": {
+        "scheduleStartDate": datetime(2015, 1, 1).date(),
+        "scheduleEndDate": datetime(2020, 1, 1).date(),
+        "startTimeOfDay": datetime.utcnow() + timedelta(minutes=2)
+    },
+    "transferSpec": {
+        'gcsDataSource': {
+            'bucketName': GCP_TRANSFER_FIRST_TARGET_BUCKET
+        },
+        "gcsDataSink": {
+            "bucketName": GCP_TRANSFER_SECOND_TARGET_BUCKET
+        },
+        "transferOptions": {
+            "overwriteObjectsAlreadyExistingInSink": True
+        },
+    }
+}
+# [END howto_operator_gct_create_job_body_gcp]
+
+# [START howto_operator_gct_update_job_body]
+update_body = {
+    "projectId": GCP_PROJECT_ID,
+    "transferJob": {
+        "description": "%s_updated" % GCP_DESCRIPTION
```

Comment:
This is my private preference.  Formerly (when I was learning Python) it was the most common way of formatting text. Formerly (when I was learning Python) it was the most common way of formatting text. We can think about what to do about it, but there was a change in the whole project.

Related: https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-6+Enforce+the+usage+of+a+code+formatter

Reply:
It's not a major issue, however I think most GCP operators (at least those developed by us last quarter) use `.format()`, so it might be good to have a single standard.

Reply:
I checked this and in fact the format method is used in other places. @antonimaciej  Can you correct this?